mono/metadata/metadata.c

```diff
@@ -2426,7 +2430,40 @@ dump_ginst (MonoGenericInst *ginst)
 	g_print (">");
 }*/
 
+static gboolean
+aggregate_modifiers_equal (gconstpointer ka, gconstpointer kb)
```

Comment:
At some point all of this modifier stuff probably deserves it's own file, outside of metadata. The churn might be too much for this PR though.